infor-design/enterprise
# 2919 - IE 11 Dropdowns in Datagrid Editing

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

A workaround added on https://github.com/infor-design/enterprise/pull/2675 caused some instances of the dropdown in a datagrid cell to no longer selct values. This issue "almost" seemed important enough for a patch or add to 4.23 but no one asked so leaving it for version next for now.

**Related github/jira issue (required)**:
Fixes #2919 
Related to #2526

**Steps necessary to review your pull request (required)**:

**Test One**
- go to http://localhost:4000/components/datagrid/example-editable.html
- open the dropdowns in the last column and select values
- values should stay selected

**Test Two**
- go to http://localhost:4000/components/datagrid/test-toggle-editable.html
- press edit to enable editing
- open the dropdowns in the last column and select values
- values should stay selected

**Test Three (Retesting 2526)**
- go to http://localhost:4000/components/datagrid/test-filter-load-data-update-columns.html
- open the filter drop down
- pick a value
- click out and close
- open filter then select Place On-Hold, then close dropdown
- open filter again, deselect Place On-Hold then select Request Review, close dropdown and the values should be in sync...
- open the page in IE 11 - will now load

**Included in this Pull Request**:
